.github/workflows/code-review.yml

```diff
@@ -0,0 +1,22 @@
+name: Code Review
+
+on:
+  pull_request: # Filter out draft pull requests
```

Comment:
Good call on filtering out the draft PRs - we do take advantage of those pretty regularly.